src/eddymotion/model/base.py

```diff
@@ -28,13 +28,21 @@
 from dipy.core.gradients import gradient_table
 from joblib import Parallel, delayed
 
+a_min_S0 = 1e-5
+a_max_S0 = 1.0
+bval_upper_cap = 1000
+bval_lower_th = 50
+bval_upper_th = 10000
+percentil_percentage = 75
+timeframe_midpoint_tol = 1e-2  # in seconds
```

Comment:
I declared this variables at the global scope, but should they be made method parameters ?

Reply:
These are constants, so I would name them in all caps and set some clear prefix for all of them:

```suggestion
DEFAULT_MIN_S0 = 1e-5
DEFAULT_MAX_S0 = 1.0
DEFAULT_MAX_BVALUE = 1000
DEFAULT_LOWB_THRESHOLD = 50
DEFAULT_HIGHB_THRESHOLD = 10000
DEFAULT_CLIP_PERCENTILE = 75
DEFAULT_TIMEFRAME_MIDPOINT_TOL = 1e-2  # in seconds
```

Reply:
Done in 324d2ee.